scst-store/amr-graphql-query.hbs.md

```diff
@@ -0,0 +1,255 @@
+# AMR Graphql Querying
```

Comment:
A couple of structural things:
1. there is a `amr` folder which this file is not in. - https://github.com/pivotal/docs-tap/tree/main/scst-store/amr
2. This file isn't linked to anywhere that I can tell (ie the table of contents) which means that this page would not be findable.

Reply:
Yep I will be rebasing from main and making some changes, putting this MR into draft for now. Thanks for taking a look!